content/applications/sales/subscriptions/reports.rst

```diff
@@ -1,120 +1,397 @@
+the title field, as well: :guilabel:`Default filter` and :guilabel:`Shared`.
 
-- **Churn Rate**: Also known as the Rate of Attrition or Customer Churn, the churn rate can be
-  defined, in this case, as the percentage of subscribers who discontinued their subscriptions
-  within a given time period. We can distinguish two types of Churn:
+Ticking the check box beside :guilabel:`Default filter` makes the newly-favorited filter the default
+option for that reporting page.
 
-  - **Logo Churn**: It corresponds to the subscription cancellation rate.
+Ticking the box beside :guilabel:`Shared` makes the newly-favorited filter available to other users
+in the database.
 
-  - **Revenue Churn**: It corresponds to the monthly recurring revenue loss rate.
+.. note::
+   The :guilabel:`Default filter` and :guilabel:`Shared` options are **not** required. Only *one* of
+   those options can be selected at a time.
 
-    .. example::
-       Let's imagine a 2$ increase in a subscription service.
+To save that filter(s), click :guilabel:`Save` in the :guilabel:`Favorites` section of the drop-down
+filter menu.
 
-       - We lost 3 customers out of the initial 20, which generates a **Logo Churn** of 15%.
-       - | Therefore, the 56$ of MRR difference out of the initial 600$ causes a **Revenue Churn**
-         | of 9,33%.
+When clicked, that saved filter appears beneath the :guilabel:`Favorites` column of the drop-down
+filter menu, and a :guilabel:`⭐ (gold star)` icon appears beside the favorite filter's name in the
+search bar.
 
-       .. image:: reports/difference-between-logo-churn-and-revenue-churn.png
-          :align: center
-          :alt: Difference between logo churn and revenue churn in Odoo Subscriptions
+Views
+-----
 
-       Reminder: even though they seem to evolve in the same direction most of the time, it might
-       not be the case all the time.
+On the :guilabel:`Subscription Analysis`, :guilabel:`MRR Breakdown`, and :guilabel:`MRR Analysis`
+reporting pages, three different view options are located in the upper-right corner.
 
-- **Customer Lifetime Value (CLV)**: Indicates how much revenue can be expected for a customer
-  during his/her entire contract. This approach emphasizes the importance of customer retention,
-  shifting our focus from a quarterly or yearly approach to a long-term one.
+.. note::
+   There are *no* other view options available on the :guilabel:`Retention Analysis` reporting page.
 
-Check out the different kinds of reports you can access from the **Odoo Subscriptions**
-application.
+The available view options, from left to right, are:
 
-Subscriptions analysis report
-=============================
+.. image:: reports/subscriptions-analysis-page-view-options.png
+  :align: center
+  :alt: The different view options available on the Subscriptions Analysis page.
+
+- :guilabel:`Graph`
+- :guilabel:`Pivot`
+- :guilabel:`List`
```

Comment:
Suggesting swapping the position of the image with the bullet points, so that there is not this awkward whitespace:
<img width="603" alt="Screenshot 2024-02-07 at 10 18 31 AM" src="https://github.com/odoo/documentation/assets/36018073/073a77f2-d5e0-4543-9560-75acb6f3501a">
```suggestion
- :guilabel:`Graph`
- :guilabel:`Pivot`
- :guilabel:`List`

.. image:: reports/subscriptions-analysis-page-view-options.png
   :align: center
   :alt: The different view options available on the Subscriptions Analysis page.
```